lib/senkyoshi/models/questions/matching.rb

```diff
@@ -9,6 +9,16 @@ def initialize
       @distractors = []
     end
 
+    def strip_select_html(text)
+      text.gsub(/<\/?p[^>]*>/i, "").          # <p> tags
+        gsub(/<\/?b[^>]*>/i, "").             # <b> tags
+        gsub(/<\/?strong[^>]*>/i, "").        # <strong> tags
+        gsub(/<\/?em[^>]*>/i, "").            # <em> tags
+        gsub(/<\/?span[^>]*>/i, "").          # <span> tags
+        gsub(/<\/?i(?!mg)[^>]*>/i, "").       # <i> tags, ignores <img> tags
+        gsub(/ style=("|')[^"']*("|')/i, "")  # inline styles
+    end
```

Comment:
Any suggestions on additions or deletions for this list of HTML tags would be welcome.